### DISCUSSION TITLE
1.5.0 release date

### DISCUSSION THREAD

**Discussion by srinisubramanian** [apache/superset#19376] [General]
Any ETA being discussed for release of 1.5.0?

**Comment by villebro** [comment]
Hi @srinisubramanian , I'm going to be release manager for the 1.5 branch. Work has already started (see PRs labelled with `lts-v1`), and we're looking to have a first testing version (pre release candidate) in testing next week. If all goes well we can have the first release candidate done shortly after that.

**Reply by srinisubramanian** [reply]
That's great. Look forward to testing it

**Reply by srinisubramanian** [reply]
Hi @villebro  I noticed that PRs #17738 and #17740 fixing the crucial import/export fix are merged to master but dont have the label you are referring to .. does it mean they will not make it to 1.5.0?

**Reply by villebro** [reply]
No, we just haven't labelled them yet. The majority of the work for the pre-rc is going through all PRs that have been merged after the LTS 1.x cut that need to be cherried in. But thanks for the heads up; I'll make sure both of those are tagged for inclusion.

**Reply by villebro** [reply]
FYI @srinisubramanian I just tagged those PRs to make sure I don't forget

**Reply by srinisubramanian** [reply]
Great. Appreciate your help in this regard. Cheers
